PayvoHQ/wallet
# fix: update welcome modal titles per step

## Summary

https://app.clickup.com/t/1da2z81

Now the title of every step changes 

**To test:**
1. add a new user so you see the welcome modal
2. navigate through the steps
3. see the titles in the modal

## Checklist
